### ISSUE TITLE
Create permanent Safe App for Omen on xDai Gnosis Safe App page

### ISSUE THREAD

**Issue by skyminert** [protofire/omen-exchange#1877]
Currently, a user can manually add Omen.eth.link as a Custom Safe App in Gnosis Safe on xDai.

Ideally, we could create a permanent Omen App that shows for all users of Gnosis Safe.  

https://xdai.gnosis-safe.io/app

I think there is probably an approval process by Gnosis to get it "whitelisted".  

https://docs.gnosis.io/safe/docs/sdks_safe_apps/#get-your-safe-app-listed-in-the-safe-multisig

**Comment by skyminert** [comment]
Could add this to Gnosis Safe mainnet App page too if appropriate.

**Comment by KadenZipfel** [comment]
> Could add this to Gnosis Safe mainnet App page too if appropriate.

Yeah, imo there's probably a lot more mainnet safes, and mainnet safes could still interact with xdai markets through the relay so would be good to have both networks supported.
